content/beginner/160_advanced-networking/secondary_cidr/cleanup.md

```diff
@@ -25,11 +26,11 @@ kubectl edit daemonset -n kube-system aws-node
         - name: MY_NODE_NAME
 ...
 ```
-Delete custom resource objects from ENIConfig CRD
+Delete custom resource objects from ENIConfig CRD. Replace yaml filenames based on your region and AZs.
 ```
-kubectl delete eniconfig/group1-pod-netconfig
-kubectl delete eniconfig/group2-pod-netconfig
-kubectl delete eniconfig/group3-pod-netconfig
+kubectl delete -f eniconfig
```

Comment:
This assumes that the user is in the correct folder. If you're going to reference a local file or directory, please add a `cd` statement to the first line to ensure they are in the correct location.

Reply:
Applied fix: 3baf530